homeworks/vladyslav.makedon_Vladmaked/8-components-templates/Gallery-app/src/app/components/gallery-item/gallery-item.component.ts

```diff
@@ -0,0 +1,36 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {Images} from '../../images';
+
+@Component({
+  selector: 'app-gallery-item',
+  templateUrl: './gallery-item.component.html',
+  styleUrls: ['./gallery-item.component.css']
+})
+export class GalleryItemComponent implements OnInit {
+
+  @Input() image!: Images;
+  @Input() selectedIndex: number = 0;
+
+  @Output() isLoadedImageEvent = new EventEmitter<boolean>();
+  @Output() selectedImageEvent = new EventEmitter<Images>();
+  @Output() selectedImageIndexEvent = new EventEmitter<number>();
+  public isLoadedImage: boolean = false;
```

Comment:
Don't use the `public` modifier for any fields and methods.

Why? This creates a visual noise that distracts you from private fields.